### PR TITLE
More mnemonics for icarus/random wallets

### PR DESCRIPTION
**Pull request by paweljakubas**
# Issue Number

#1803 

# Overview

- [x] I have enabled more mnemonics lengths for random/icarus wallets
- [x] I have updated swagger and Malformed in core unit tests
- [x] I have updated byron integration scenarios

# Comments

### CODE REVIEW

**Comment by paweljakubas** [comment]
bors try

File: lib/core-integration/src/Test/Integration/Scenario/API/Byron/Wallets.hs [hunk 1]
```diff
@@ -221,17 +221,17 @@ spec = do
...
         it' "icarus" (genMnemonics @9)  scenarioFailure -- ❌
-        it' "icarus" (genMnemonics @12) scenarioFailure -- ❌
+        it' "icarus" (genMnemonics @12) scenarioSuccess -- ✔️
         it' "icarus" (genMnemonics @15) scenarioSuccess -- ✔️
-        it' "icarus" (genMnemonics @18) scenarioFailure -- ❌
-        it' "icarus" (genMnemonics @21) scenarioFailure -- ❌
-        it' "icarus" (genMnemonics @24) scenarioFailure -- ❌
+        it' "icarus" (genMnemonics @18) scenarioSuccess -- ✔️
+        it' "icarus" (genMnemonics @21) scenarioSuccess -- ✔️
+        it' "icarus" (genMnemonics @24) scenarioSuccess -- ✔️
```
Comment (KtorZ) [inline lib/core-integration/src/Test/Integration/Scenario/API/Byron/Wallets.hs:234]: Seems about right :heavy_check_mark:

**Review by KtorZ — APPROVED** [review]
LGTM. Thanks @paweljakubas

**Pull request merged**